packages/frontend-2/components/auth/LoginPanel.vue

```diff
@@ -63,7 +63,7 @@ const props = withDefaults(
   }>(),
   {
     dialogMode: false,
-    title: 'Speckle Login',
+    title: 'Speckle login',
     subtitle: 'Connectivity, Collaboration and Automation for 3D'
```

Comment:
Wasn't sure if we wanted to change this tagline, or we use it across different places with this formatting

Reply:
Yeah let's just keep this as is for now